ansible-ipi-install/inventory/hosts.sample

```diff
@@ -92,6 +92,12 @@ prov_ip=172.22.0.3
 # By default this feature is enabled and set to true. Uncomment below to disable and use IPMI.
 #redfish_inspection=false
 
+# (Optional) Path to a fake root and appends them into ignition configuration.
+# If defined the customize_filesystem playbook will look for files in master and worker subdirs.
+# Otherwise it will look in {{ role_path }}/files/customize_filesystem
+# If no files are found then no action will be performed.
+#customize_ironic_path="path/to/fake/root"
```

Comment:
Any way we can think of a more representative name? I just think `customize_ironic_path` is confusing :)

Reply:
How about filesystem_override_path?

Reply:
How about we change it to https://openshift-kni.github.io/baremetal-deploy/latest/Ansible%20Playbook%20Install.html#ansible-playbook-customizing-the-node-filesystems

```
# Modify files on the node filesystems, you can augment the "fake" roots for the control plane and worker nodes.
# If defined, playbook will look for files in control plane and worker subdirectories.
# Otherwise, it will look in {{ role_path }}/files/customize_filesystem (default)
# For more information on modifying node filesystems visit: https://bit.ly/36tD30f
customize_node_filesystems="/path/to/customized/filesystems"
```

Reply:
I like it!